README.md

```diff
@@ -1 +1,297 @@
-# traceability-viewer
\ No newline at end of file
+# traceability-viewer
+
+## Table of Contents
+
+- [Getting Started](#Getting-Started)
+- [Configuration](#Configuration)
+  - [traceability_export](#traceability_export)
+  - [html_documentation_root](#html_documentation_root)
+  - [layered](#layered)
+  - [layers](#layers)
+  - [item_colors](#item_colors)
+  - [link_colors](#link_colors)
+  - [backwards_relationships](#backwards_relationships)
+  - [visualised_properties](#visualised_properties)
+- [Usage](#Usage)
+
+
+## Getting Started
+
+First, you need to install docker compose: https://docs.docker.com/compose/install/linux/#install-the-plugin-manually
+
+<!-- https://docs.docker.com/engine/reference/commandline/compose_up/ -->
+
+After installation, copy the JSON file that contains the data to the project folder and copy the `.env.example` to `.env`.
+```
+# copy traceability_export.json file to project folder
+cp /path/to/traceability_export.json .
+# copy example .env to your .env
+cp .env.example .env
+```
+Lastly, use docker compose to start the application.
+
+```
+docker compose up --build --remove-orphans
+```
+
+## Configuration
+
+The configuration takes the form of a YAML file. The [config.yml](config.yml) is an example that requires modification.
+
+### traceability_export
+
+> [!IMPORTANT]
+> `traceability_export` is a required configuration parameter.
+
+This is the path to the JSON file that contains all the data. The items in the JSON file needs to have a name and targets.
+
+It should be structured as (this is the minimum required):
+```
+[
+    {
+        "name" : "<name-of-item>",
+        "targets": {
+            "<relation-type-1>": [
+                "<target-item-name-1>",
+                "<target-item-name-2>",
+                ...
+            ],
+            "<relation-type-2>": [
+                "<target-item-name-3>",
+                "<target-item-name-4>",
+                ...
+            ],
+        }
+    },
+    ...
+]
+```
+
+When you use the [traceability-plugin](https://melexis.github.io/sphinx-traceability-extension/index.html), you can [export](https://melexis.github.io/sphinx-traceability-extension/configuration.html#export) the documentation items as a JSON.
+
+### html_documentation_root
+
+> [!NOTE]
+> `html_documentation_root` is not a required configuration parameter.
+
+The html root to the documentation of that item. This can also be the path to a local html folder.
+
+This is only workable when you use the exported JSON of the [traceability-plugin](https://melexis.github.io/sphinx-traceability-extension/configuration.html#export-to-json).
```

Comment:
I think it is only useful when you use the traceability plugin, but actually it can be any documentation that is composed of attributes of that element

Reply:
The JSON database must contain the path to the RST document and the item ID/name must be appended to the HTML filename with a `#`. The traceability plugin does both.

```suggestion
This configuration parameter can only be used if [the JSON database](###traceability_export) contains a ``"document"`` attribute with the relative path to the reST file in which the item is defined,
and if the item ID/name is appended to the HTML filename with a `#`. 
Our [traceability-plugin](https://melexis.github.io/sphinx-traceability-extension/configuration.html#export-to-json) does both.
```

Reply:
Suggestion applied in a7001e19f8f006e609f1a4d6a711a01590a42923